packages/design-system/src/Checkbox/Checkbox.styles.tsx

```diff
@@ -1,12 +1,13 @@
 import styled, { css } from "styled-components";
 
 const Variables = css`
+  --ads-v2-colors-control-checkbox-icon-checked-fg: var(
```

Comment:
Here, can we have the component level token consume the category token? It gets confusing when they are both in the same file.

Reply:
Resolving this as we discussed why this has done.

Reply:
Documenting our decision: We decided to let this be because we are only creating component-level tokens when that token is likely to change over the various states or variants of the component. Since the category token `--ads-v2-colors-control-checkbox-icon-checked-fg` is used only in one place, we've let it be.